modules/rules_wasm/1.0.14/presubmit.yml

```diff
@@ -0,0 +1,17 @@
+matrix:
+  platform:
+    - debian10
+    - ubuntu2004
+    - macos
+    - macos_arm64
+  bazel:
+    - "6.x"
+    - "7.x"
+tasks:
+  run_all_tests:
+    name: Run all tests
+    platform: ${{ platform }}
+    bazel: ${{ bazel }}
+    test_targets:
+      - "//..."
```

Comment:
By default, the presubmit sets up a dummy module which depends on this module to build & test.

If you want to test this module as a root module, you can follow https://github.com/bazelbuild/bazel-central-registry/blob/main/docs/README.md#test-module and set `module_path: "."`

Reply:
Ah, I didn't read the docs closely enough :) I load my test module as a `dev_dependency` into the root module, because I have a sanity-check test in the root module that runs a test from the test module and compares stdout to an expected value. I did this just in case the example test fails to actually run for whatever reason, yet the command still exits with status zero somehow.

So these results would make sense, because the `dev_dependency` would be ignored if the root module is itself being loaded by a dummy module.

I have GH actions on my repo to exercise the root module tests on both Linux and Mac, so I'll just exercise the test module here.